framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -213,7 +242,7 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 				break ccmInEventsCheck;
 			}
 
-			const { modules } = await this._sidechainAPIClient.invoke<{ modules: ModuleMetadata }>(
+			const { modules } = await this._sendingChainClient.invoke<{ modules: ModuleMetadata }>(
 				'system_getMetadata',
 			);
 			const interoperabilityMetadata = modules.find(m => m.name === MODULE_NAME_INTEROPERABILITY);
```

Comment:
In below, it's using `interoperabilityModule?`, but it seems always expected, so we can actually check if it's not undefined, and throw error